tests/helpers/cucumber-hooks.conf.js

```diff
@@ -11,16 +11,18 @@ exports.config = Object.assign({}, config, {
         browser.pause(30)
         browser.Cucumber_Test = 1
     },
-    beforeStep: async function (uri, feature, stepData, context) {
+    beforeStep: async function (fileName, doc, step, world) {
```

Comment:
I think before/after Step/Hook functions should have args like this:

1. test (for cucumber it's an object containing uri, feature, step, ...)
2. test context (cucumber world)
3. [only after* hooks] webdriverio object containing result, duration and other fields (we may move title, fullTitle and some other common fields here as well)